homeassistant/components/camera/util.py

```diff
@@ -0,0 +1,29 @@
+"""Camera utility functions."""
```

Comment:
In Home Assistant normally utils are tools that are not aware of Home Assistant and helpers are tools that are aware of Home Assistant.